website/docs/cli/auth/index.html.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: "docs"
 page_title: "Authentication - Terraform CLI"
+description: "Documentation about the `login` and `logout` commands that help automate getting an API token for your Terraform Cloud account."
```

Comment:
I don't think our website framework can support formatting sequences like backticks in the `description` field, because it ultimately ends up being the value of a `meta` element in the final HTML, so I think we've typically written literal names like these in quotes in the description context as an alternative to ensure that the result "looks right". :thinking:

Reply:
@apparentlymart thank you for flagging :-) I just pushed changes that do this in another PR: https://github.com/hashicorp/terraform/pull/29117.  They built okay for me locally but now I'm nervous - do the docs in that PR look okay to you?

Reply:
Okay note that I spoke with the @jescalan on the web dev team, and they said that if I built the website locally and everything looks fine then I'm okay to leave it. BUT I will be mindful of this :-) Thank you for flagging!

Reply:
I agree with @apparentlymart here honestly, if I recall we discussed this as well, but backtick formatting will not render correctly in meta descriptions and are better left out or using normal quotes if it's not too much work to replace them!

Reply:
ok - I'll go ahead and do that, thanks!